tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -50,6 +50,9 @@ sealed trait NativeConfig {
   /** Shall we optimize the resulting NIR code? */
   def optimize: Boolean
 
+  /** Shall we use the incremental compilation? */
+  def incrementalCompilation: Boolean
```

Comment:
The name for boolean variable should be a verb, instead of a noun. Probably the safest choice in this case would be `{use,enable}IncrementalCompilation` or `compileIncrementally`